<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>994</width>
    <height>504</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QChartView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>10</y>
      <width>541</width>
      <height>451</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>421</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>Data Selector</string>
    </property>
    <widget class="QWidget" name="horizontalLayoutWidget_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>401</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QProgressBar" name="progressBar">
        <property name="value">
         <number>0</number>
        </property>
        <property name="textVisible">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="load_button">
        <property name="text">
         <string>Load</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cancle_button">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>401</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QCheckBox" name="order_id_check">
        <property name="text">
         <string>Order ID</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="fee_check">
        <property name="text">
         <string>Fee</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>401</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QCheckBox" name="dep_info_check">
        <property name="text">
         <string>Departure Info</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="dest_info_check">
        <property name="text">
         <string>Destination Info</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>421</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>Parameter Chooser</string>
    </property>
    <widget class="QWidget" name="horizontalLayoutWidget_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>401</width>
       <height>21</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Start Time</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>End Time</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Time Step</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>401</width>
       <height>25</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <widget class="QDateTimeEdit" name="start_time">
        <property name="maximumDateTime">
         <datetime>
          <hour>23</hour>
          <minute>0</minute>
          <second>0</second>
          <year>2016</year>
          <month>11</month>
          <day>15</day>
         </datetime>
        </property>
        <property name="minimumDateTime">
         <datetime>
          <hour>0</hour>
          <minute>0</minute>
          <second>0</second>
          <year>2016</year>
          <month>11</month>
          <day>1</day>
         </datetime>
        </property>
        <property name="currentSection">
         <enum>QDateTimeEdit::MonthSection</enum>
        </property>
        <property name="displayFormat">
         <string>MM-dd H</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateTimeEdit" name="end_time">
        <property name="maximumDateTime">
         <datetime>
          <hour>23</hour>
          <minute>0</minute>
          <second>0</second>
          <year>2016</year>
          <month>11</month>
          <day>15</day>
         </datetime>
        </property>
        <property name="minimumDateTime">
         <datetime>
          <hour>0</hour>
          <minute>0</minute>
          <second>0</second>
          <year>2016</year>
          <month>11</month>
          <day>1</day>
         </datetime>
        </property>
        <property name="currentSection">
         <enum>QDateTimeEdit::MonthSection</enum>
        </property>
        <property name="displayFormat">
         <string>MM-dd H</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="step_box">
        <item>
         <property name="text">
          <string>10min</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>30min</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1h</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2h</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget_9">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>401</width>
       <height>24</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QComboBox" name="graph_box"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>401</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_9">
      <item>
       <widget class="QPushButton" name="choosegrid">
        <property name="text">
         <string>Choose Grids</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>421</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>Analysis</string>
    </property>
    <widget class="QPushButton" name="heat_button">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>401</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Draw Thermal Diagram for the Selected Grids </string>
     </property>
    </widget>
    <widget class="QPushButton" name="plot_button">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>401</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Update Plot</string>
     </property>
    </widget>
    <widget class="QPushButton" name="route_button">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>401</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Find Route</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>994</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QGraphicsView</extends>
   <header location="global">QtCharts</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
